src/ClientRuntime/Microsoft.Azure.Common/project.json

```diff
@@ -36,13 +36,10 @@
         "System.Net.Http": "",
         "System.Net.Http.WebRequest": "",
         "System.Runtime.Serialization": "",
-        "System.Threading": "",
```

Comment:
I have a repro in VS2015 targeting net45.  Just adding a package with the nuget dependency  causes the nuget ui to fail with the message described in the referenced issue.  The compiled dll doesn't actually depend on any of these, so it is correct one way or the other, regardless